crates/test_utils/src/main.rs

```diff
@@ -10,8 +10,8 @@ use test_utils::connector_auth::{ConnectorAuthType, ConnectorAuthenticationMap};
 // Just by the name of the connector, this function generates the name of the collection
 // Example: CONNECTOR_NAME="stripe" -> OUTPUT: postman/stripe.postman_collection.json
```

Comment:
Can we update the documentation comment as well? Maybe something like so:

```suggestion
// Example: CONNECTOR_NAME="stripe" -> OUTPUT: postman/stripe
```

Reply:
Thanks.  Addressed !!